src/po/uk.cp1251.po

```diff
@@ -12,18 +12,18 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: vim 8.1\n"
+"Project-Id-Version: vim 9.0\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2022-05-02 17:40+0300\n"
-"PO-Revision-Date: 2022-05-20 17:12+0300\n"
+"POT-Creation-Date: 2023-08-24 11:13+0300\n"
+"PO-Revision-Date: 2023-08-25 13:12+0300\n"
 "Last-Translator: ������� ������ <sakhnik@gmail.com>\n"
 "Language-Team: Ukrainian\n"
 "Language: uk\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=cp1251\n"
+"Content-Type: text/plain; charset=UTF-8\n"
```

Comment:
This looks wrong.
`uk.cp1251.po` should be automatically updated by running `make` in the `src/po/` directory.

Reply:
It didn't, by the way. But I've updated it manually. Thanks!

Reply:
> It didn't, by the way.

Hmm. If `uk.cp1251.po` is newer than `uk.po`, `make` will not update it.
I'm not sure if this is the case, though.

Reply:
All right, that was my manual intervention that messed things up. When I just edit uk.po and leave uk.cp1251.po alone, everything works fine. I just need to remember this.